Source/DafnyCore/Resolver/ModuleResolver.cs

```diff
@@ -1162,43 +1162,33 @@ public partial class ModuleResolver {
         CallGraphBuilder.Build(declarations, reporter);
       }
 
+      // The call graph hasn't been completely constructed, because it's missing the edges having to do with
```

Comment:
Seems like an ordering issue. Can you explain why the order is the way it is?

Reply:
The postconditions and bodies of prefix lemmas have not been filled in yet. That's done by a call to `FillInPostConditionsAndBodiesOfPrefixLemmas`, which occurs later. That method depends on the call graph having been filled in with everything else. I looked through the code of `FillInPostConditionsAndBodiesOfPrefixLemmas` and the visitors it calls, but could not find any other dependencies. So, it is possible that one could move the call to `FillInPostConditionsAndBodiesOfPrefixLemmas` here, after which point the call graph would be fully constructed.

This would be good to try out, but I suggest such an investigation be done separately from this PR.

Reply:
I'm trying it out in https://github.com/dafny-lang/dafny/pull/5098. We'll see if it works.